lib/hubspot/ticket.rb

```diff
@@ -0,0 +1,61 @@
+require 'hubspot/utils'
+
+module Hubspot
+  #
+  # HubSpot Tickets API
+  #
+  # {https://developers.hubspot.com/beta-docs/guides/api/crm/objects/tickets}
+  #
+  class Ticket
+    TICKETS_PATH = '/crm/v3/objects/tickets'
+    TICKET_PATH  = '/crm/v3/objects/tickets/:ticket_id'
+
+    attr_reader :properties, :id
+
+    def initialize(response_hash)
+      @id = response_hash['id']
+      @properties = response_hash['properties'].deep_symbolize_keys
+    end
+    class << self
```

Comment:
```suggestion
    end

    class << self
```
please run rubocop on this file :bow: